tests/cmd/builds/commands.go

```diff
@@ -18,17 +18,22 @@ import (
 const ExampleImage = "deis/example-dockerfile-http"
 
 // Create executes `deis builds:create` as the specified user.
-func Create(user model.User, app model.App) {
-	createOrPull(user, app, "builds:create")
+func Create(user model.User, app model.App, opts ...string) {
+	createOrPull(user, app, "builds:create", opts)
 }
 
 // Pull executes the `deis pull` shortcut as the specified user.
-func Pull(user model.User, app model.App) {
-	createOrPull(user, app, "pull")
+func Pull(user model.User, app model.App, opts ...string) {
+	createOrPull(user, app, "pull", opts)
 }
 
-func createOrPull(user model.User, app model.App, command string) {
-	sess, err := cmd.Start("deis %s --app=%s %s", &user, command, app.Name, ExampleImage)
+func createOrPull(user model.User, app model.App, command string, opts []string) {
+	image := opts[0]
```

Comment:
I'm not quite sure this is a good way to send an option list to `createOrPull`. How about a `struct` instead?

```
type CreateOrPullOpts struct {
    Image string
}
```

That way we can directly reference them instead of having to keep track of where certain values are stored in a slice.